.changeset/fancy-coins-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+allow single spaces and newline characters in useConsistentCurlyBraces rule #4154
```

Comment:
```suggestion
Allowed single spaces and newline characters in `useConsistentCurlyBraces` rule.
```

Reply:
The Guide says 

> Use the present tense
> – https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#writing-a-changeset

this is now past tense.

Reply:
Good catch! I’ll update the guide.